src/components/header/SearchNavbar.jsx

```diff
@@ -4,20 +4,16 @@ import SearchIcon from '@mui/icons-material/Search';
 import FlipCameraAndroidIcon from '@mui/icons-material/FlipCameraAndroid';
 import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
 import ShoppingCartOutlinedIcon from '@mui/icons-material/ShoppingCartOutlined';
-// import { useDispatch, useSelector } from 'react-redux';
-// import { cartAction } from '../../store/cart-slice';
```

Comment:
Hey, @debbyuzuegbu   Pls  uncomment all the redux store action you comment out. because the state is being used here